### ISSUE TITLE
v6.1, cuda11.0，export onnx model, break?

### ISSUE THREAD

**Issue by xinsuinizhuan** [ultralytics/yolov5#6883]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

C:\ProgramData\Anaconda3\envs\yolov5\python.exe E:/Item/Item_done/yolo/yolo5/yolov5_v6.1/yolov5/export.py --weights=yolov5s.pt --dynamic --include=onnx --opset=11
export: data=data\coco128.yaml, weights=['yolov5s.pt'], imgsz=[640, 640], batch_size=1, device=cpu, half=False, inplace=False, train=False, optimize=False, int8=False, dynamic=True, simplify=False, opset=11, verbose=False, workspace=4, nms=False, agnostic_nms=False, topk_per_class=100, topk_all=100, iou_thres=0.45, conf_thres=0.25, include=['onnx']
YOLOv5  v6.1-11-g63ddb6f torch 1.7.1+cu110 CPU

Fusing layers... 
Model Summary: 213 layers, 7225885 parameters, 0 gradients, 16.5 GFLOPs

PyTorch: starting from yolov5s.pt with output shape (1, 25200, 85) (14.8 MB)

ONNX: starting export with onnx 1.10.2...

进程已结束,退出代码-1073741819 (0xC0000005)

### Minimal Reproducible Example

(yolov5) E:\Item\Item_done\yolo\yolo5\onnx_model_export>pip list
Package                 Version
----------------------- ---------------------
absl-py                 1.0.0
addict                  2.4.0
cachetools              4.2.4
certifi                 2021.10.8
charset-normalizer      2.0.9
click                   8.0.3
colorama                0.4.4
configparser            5.2.0
coremltools             5.1.0
cycler                  0.11.0
defusedxml              0.7.1
docker-pycreds          0.4.0
editdistance            0.6.0
fast_ctc_decode         0.3.0
flatbuffers             2.0
fonttools               4.28.5
future                  0.18.2
gitdb                   4.0.9
GitPython               3.1.24
google-auth             2.3.3
google-auth-oauthlib    0.4.6
grpcio                  1.43.0
hyperopt                0.1.2
idna                    3.3
imageio                 2.13.5
importlib-metadata      4.10.0
joblib                  1.1.0
jstyleson               0.0.2
kiwisolver              1.3.2
Markdown                3.3.6
matplotlib              3.5.1
mpmath                  1.2.1
networkx                2.6.3
nibabel                 3.2.1
nltk                    3.6.7
numpy                   1.19.5
oauthlib                3.1.1
onnx                    1.10.2
onnx-simplifier         0.3.6
onnxoptimizer           0.2.6
onnxruntime             1.10.0
onnxruntime-gpu         1.10.0
opencv-contrib-python   4.5.4+dummy
opencv-python           4.5.4.60
openvino                2021.4.2
openvino-dev            2021.4.2
packaging               21.3
pandas                  1.1.5
parasail                1.2.4
pathtools               0.1.2
Pillow                  8.4.0
pip                     21.3.1
progress                1.6
promise                 2.3
protobuf                3.19.1
psutil                  5.8.0
py-cpuinfo              8.0.0
pyasn1                  0.4.8
pyasn1-modules          0.2.8
pydicom                 2.2.2
pymongo                 4.0.1
pyparsing               3.0.6
python-dateutil         2.8.2
pytz                    2021.3
PyWavelets              1.2.0
PyYAML                  6.0
rawpy                   0.17.0
regex                   2021.11.10
requests                2.26.0
requests-oauthlib       1.3.0
rsa                     4.8
scikit-image            0.19.1
scikit-learn            1.0.2
scipy                   1.5.4
seaborn                 0.11.2
sentencepiece           0.1.96
sentry-sdk              1.5.1
setuptools              60.1.0
Shapely                 1.8.0
shortuuid               1.0.8
six                     1.16.0
smmap                   5.0.0
subprocess32            3.5.4
sympy                   1.9
tensorboard             2.7.0
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.0
tensorrt                0.0.1
termcolor               1.1.0
texttable               1.6.4
thop                    0.0.31.post2005241907
threadpoolctl           3.0.0
tifffile                2021.11.2
tokenizers              0.11.2
torch                   1.7.1+cu110
torchaudio              0.7.2
torchvision             0.8.2+cu110
tqdm                    4.62.3
typing_extensions       4.0.1
urllib3                 1.26.7
wandb                   0.12.9
Werkzeug                2.0.2
wheel                   0.37.1
yamlloader              1.1.0
yaspin                  2.1.0
zipp                    3.6.0

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@xinsuinizhuan it appears you may have environment problems. Please ensure you meet all dependency requirements if you are attempting to run YOLOv5 locally.  If in doubt, create a new virtual [Python 3.9](https://www.python.org/) environment, clone the latest repo ([code changes daily](https://github.com/ultralytics/yolov5/commits/master)), and `pip install` [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) again from scratch.

💡 ProTip! Try one of our verified environments below if you are having trouble with your local environment.


## Requirements

[**Python>=3.7.0**](https://www.python.org/) with all [requirements.txt](https://github.com/ultralytics/yolov5/blob/master/requirements.txt) installed including [**PyTorch>=1.7**](https://pytorch.org/get-started/locally/). To get started:
```bash
git clone https://github.com/ultralytics/yolov5  # clone
cd yolov5
pip install -r requirements.txt  # install
```

[Models](https://github.com/ultralytics/yolov5/tree/master/models) and [datasets](https://github.com/ultralytics/yolov5/tree/master/data) download automatically from the latest YOLOv5 [release](https://github.com/ultralytics/yolov5/releases) when first requested.

## Environments

YOLOv5 may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Google Colab and Kaggle** notebooks with free GPU: <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>


## Status

![CI CPU testing](https://github.com/ultralytics/yolov5/workflows/CI%20CPU%20testing/badge.svg)

If this badge is green, all [YOLOv5 GitHub Actions](https://github.com/ultralytics/yolov5/actions) Continuous Integration (CI) tests are currently passing. CI tests verify correct operation of YOLOv5 training ([train.py](https://github.com/ultralytics/yolov5/blob/master/train.py)), validation ([val.py](https://github.com/ultralytics/yolov5/blob/master/val.py)), inference ([detect.py](https://github.com/ultralytics/yolov5/blob/master/detect.py)) and export ([export.py](https://github.com/ultralytics/yolov5/blob/master/export.py)) on MacOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by xinsuinizhuan** [comment]
OK.When i create a new virtual Python 3.9, it works.Thank you!

**Comment by glenn-jocher** [comment]
@xinsuinizhuan great!
